plugins/adminhelp.sma

```diff
@@ -13,133 +13,181 @@
 
 #include <amxmodx>
 
-new g_cvarDisplayClientMessage;
-new g_cvarHelpAmount;
+const MaxMapLength         = 32;
```

Comment:
Well I was mistaken because changelevel uses 64 chars. However if you look at  

https://github.com/dreamstalker/rehlds/blob/94fb06f95b4e1e9f150cd4e8c0f4e0fb86e552e2/rehlds/engine/pr_cmds.cpp#L1549

you'll see it's 32 as you said, therefore I believe ```MaxMapLength``` should be 33 (or array be ```MaxMapLength + 1```). Just tested in a server and 32 char maps do work.